test/scenarios.spec.ts

```diff
@@ -285,10 +301,11 @@ suite('== Custom Handler function scenarios ==', () => {
                 return InputUtil.isValueControlIntent(input, AmazonBuiltInSlotType.DATE);
             }
 
-            function handleSetValue(input: ControlInput) {
+            async function handleSetValue(input: ControlInput, resultBuilder: ControlResultBuilder) {
```

Comment:
adding validation checks makes the function to be same as built-in `handleSetValue`, so you are basically adding an inbuilt handler again through overrides, you may have to change the test case.